roles/ansible_service_broker/defaults/main.yml

```diff
@@ -5,6 +5,14 @@ ansible_service_broker_log_level: info
 ansible_service_broker_output_request: false
 ansible_service_broker_recovery: true
 ansible_service_broker_bootstrap_on_startup: true
-# Recommended you do not enable this for now
 ansible_service_broker_dev_broker: false
+ansible_service_broker_refresh_interval: 600s
+# Recommended you do not enable this for now
 ansible_service_broker_launch_apb_on_bind: false
+
+ansible_service_broker_config_path: /etc/ansible-service-broker/config.yaml
+ansible_service_broker_user: admin
+ansible_service_broker_password: "{{ lookup('password', '/dev/null') }}"
```

Comment:
Lets move this to a `set_fact` or a `var/main.yml` entry instead so we can keep our defaults less complex.